schema/cosa/schema_doc.go

```diff
@@ -178,7 +178,7 @@ var generatedSchemaJSON = `{
       }
  },
  "$schema":"http://json-schema.org/draft-07/schema#",
- "$id":"http://github.com/coreos/coreos-assembler/blob/main/schema/v1.json",
+ "$id":"http://github.com/coreos/coreos-assembler/blob/main/v1.json.json",
```

Comment:
this..doesn't seem right..

Reply:
based on this PR: https://github.com/coreos/coreos-assembler/pull/2301 this change is correct.